src/redux/createMigrate.test.ts

```diff
@@ -107,7 +107,7 @@ describe(createMigrate, () => {
 
     const migrate = createMigrate(failingMigrations)
     await expect(migrate(vNeg1Stub, 7)).rejects.toThrowErrorMatchingInlineSnapshot(
-      `"Cannot read properties of undefined (reading 'doesNotExist')"`
```

Comment:
Is this change expected? I think this changed with node upgrade: https://github.com/valora-inc/wallet/pull/2474/files#diff-e4e3e8adba0bed5d82fcf639a7668403d76074e01230b2ddd66c3f3b28736500R110. Do you have the right node version locally?